run_topostats.py

```diff
@@ -1,9 +1,13 @@
 """Topotracing"""
```

Comment:
Should `run_topostats.py` be part of the package?

Reply:
As the steps being run had expanded outside of the initial "Topotracing" work to include the code and steps in `dnatracing.py` I moved and rename `topostats/topotracing.py` > `run_topostats.py`.

In light of your pointer about [entry points](https://amir.rachum.com/blog/2017/07/28/python-entry-points/) (see also [here](https://setuptools.pypa.io/en/latest/userguide/entry_point.html) for documentation and doing this with `setup.cfg`) I've just created #170 to add this feature.